recipes/twilio/meta.yaml

```diff
@@ -0,0 +1,213 @@
+    - twilio.rest.preview.sync
+    - twilio.rest.preview.sync.service
+    - twilio.rest.preview.sync.service.document
+    - twilio.rest.preview.sync.service.sync_list
+    - twilio.rest.preview.sync.service.sync_map
+    - twilio.rest.preview.trusted_comms
+    - twilio.rest.preview.understand
+    - twilio.rest.preview.understand.assistant
+    - twilio.rest.preview.understand.assistant.field_type
+    - twilio.rest.preview.understand.assistant.task
+    - twilio.rest.preview.wireless
+    - twilio.rest.preview.wireless.sim
+    - twilio.rest.pricing
+    - twilio.rest.pricing.v1
+    - twilio.rest.pricing.v1.messaging
+    - twilio.rest.pricing.v1.phone_number
+    - twilio.rest.pricing.v1.voice
+    - twilio.rest.pricing.v2
+    - twilio.rest.pricing.v2.voice
+    - twilio.rest.proxy
+    - twilio.rest.proxy.v1
+    - twilio.rest.proxy.v1.service
+    - twilio.rest.proxy.v1.service.session
+    - twilio.rest.proxy.v1.service.session.participant
+    - twilio.rest.serverless
+    - twilio.rest.serverless.v1
+    - twilio.rest.serverless.v1.service
+    - twilio.rest.serverless.v1.service.asset
+    - twilio.rest.serverless.v1.service.environment
+    - twilio.rest.serverless.v1.service.function
+    - twilio.rest.studio
+    - twilio.rest.studio.v1
+    - twilio.rest.studio.v1.flow
+    - twilio.rest.studio.v1.flow.engagement
+    - twilio.rest.studio.v1.flow.engagement.step
+    - twilio.rest.studio.v1.flow.execution
+    - twilio.rest.studio.v1.flow.execution.execution_step
+    - twilio.rest.sync
+    - twilio.rest.sync.v1
+    - twilio.rest.sync.v1.service
+    - twilio.rest.sync.v1.service.document
+    - twilio.rest.sync.v1.service.sync_list
+    - twilio.rest.sync.v1.service.sync_map
+    - twilio.rest.sync.v1.service.sync_stream
+    - twilio.rest.taskrouter
+    - twilio.rest.taskrouter.v1
+    - twilio.rest.taskrouter.v1.workspace
+    - twilio.rest.taskrouter.v1.workspace.task
+    - twilio.rest.taskrouter.v1.workspace.task_queue
+    - twilio.rest.taskrouter.v1.workspace.worker
+    - twilio.rest.taskrouter.v1.workspace.workflow
+    - twilio.rest.trunking
+    - twilio.rest.trunking.v1
+    - twilio.rest.trunking.v1.trunk
+    - twilio.rest.verify
+    - twilio.rest.verify.v2
+    - twilio.rest.verify.v2.service
+    - twilio.rest.verify.v2.service.rate_limit
+    - twilio.rest.video
+    - twilio.rest.video.v1
+    - twilio.rest.video.v1.composition
+    - twilio.rest.video.v1.recording
+    - twilio.rest.video.v1.room
+    - twilio.rest.video.v1.room.recording
+    - twilio.rest.video.v1.room.room_participant
+    - twilio.rest.voice
+    - twilio.rest.voice.v1
+    - twilio.rest.voice.v1.dialing_permissions
+    - twilio.rest.voice.v1.dialing_permissions.country
+    - twilio.rest.wireless
+    - twilio.rest.wireless.v1
+    - twilio.rest.wireless.v1.sim
+    - twilio.twiml
+
+about:
+  home: "https://github.com/twilio/twilio-python/"
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.md
```

Comment:
I opened a PR to this project to also package the license 
https://github.com/twilio/twilio-python/pull/489

Reply:
This is great! Once that is merged, can refer to the license file in the original package.